package.json

```diff
@@ -42,17 +49,25 @@
     "prettier": "prettier --config ./.prettierrc --list-different \"projects/**/*{.ts,.js,.json,.scss,.html}\"",
     "prettier-fix": "prettier --config ./.prettierrc --list-different \"projects/**/*{.ts,.js,.json,.scss,.html}\" --write",
     "start": "ng serve --proxy-config mock-server/proxy.backend.js",
+    "start:cds": "ng serve storefrontapp-cds-integration --proxy-config mock-server/proxy.backend.js",
```

Comment:
What do you think, does it make more sense to group all integration scripts next to each other?
I.e.:

- first we list core scripts for starting spartacus for local dev, ci, prod, etc.
- then we list cds scripts for starting cds for local dev, ci, prod, etc.
- then we list all those scripts for the next integration
- etc.

_OR_, we keep them grouped per command:
- first we list all scripts that start spartacus for various integrations
- then we list all scripts that start spartacus for ci
- then we list all scripts that do a prod build
- etc

Reply:
yes